com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -524,6 +535,15 @@ public float renderScale
             set { m_RenderScale = ValidateRenderScale(value); }
         }
 
+        /// <summary>
+        /// Gets or Sets the ShaderQuality of the current render pipeline.
+        /// </summary>
+        public ShaderQuality shaderQuality
+        {
+            get { return m_ShaderQuality; }
+            set { m_ShaderQuality = value; }
```

Comment:
exposing a set here is problematic. That means users can change it a runtime and we need to know the quality at build time because we strip it.

Remove setter fixes the issue.